test/config/integration/server_xds.bootstrap.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
random question 1: proto as some textual form also, right? Do we expect the textual form for v2 to generally be JSON? Or do you think people will provide the other textual form for proto (I think this exists but I might be making this up). FWIW this is all pretty readable which is nice. Gives me hope. :)

random question 2: Will it be possible in v2 via bootstrap to still specify everything in a single file? Like if someone wants to have a fully static config but based in proto/JSON?